mmdeploy/codebase/mmdet/deploy/object_detection_model.py

```diff
@@ -603,6 +603,38 @@ class labels of shape [N, num_det].
         dets = torch.cat([boxes, scores], dim=2)
         return dets, labels.to(torch.int32)
 
+@__BACKEND_MODEL.register_module('vacc_det')
+class VACCDetModel(End2EndModel):
+
+    def __init__(self, backend: Backend, backend_files: Sequence[str],
+                 device: str, model_cfg: Union[str, Config],
+                 deploy_cfg: Union[str, Config], **kwargs):
+        assert backend == Backend.VACC, f'only supported vacc, but give \
+            {backend.value}'
+
+        super(VACCDetModel, self).__init__(backend, backend_files, device,
+                                               deploy_cfg, **kwargs)
+        # load cfg if necessary
+        model_cfg = load_config(model_cfg)[0]
+        self.model_cfg = model_cfg
+
+    def predict(self, imgs: Tensor, *args, **kwargs) -> List:
+        """Implement forward test.
+
+        Args:
+            imgs (Tensor): Input image(s) in [N x C x H x W] format.
+
+        Returns:
+            list[Tensor]: dets of shape [N, num_det, 5] and
+                class labels of shape [N, num_det].
+        """
+        _, _, H, W = imgs.shape
+        outputs = self.wrapper({self.input_name: imgs})
+        for key, item in outputs.items():
+            if item is None:
+                return torch.zeros(1, 0, 5), torch.zeros(1, 0)
+        
+        return torch.zeros(1, 0, 5), torch.zeros(1, 0)
```

Comment:
no real outputs returned?

Reply:
Because vacc backend don't support postprocessing，only get feature map now

Reply:
OK. So, we can do postprocessing in python just like other backends. [This](https://github.com/open-mmlab/mmdeploy/blob/8a050f10dc9be49168ff80f9dfdbb25ae0fcb61d/mmdeploy/codebase/mmdet/deploy/object_detection_model.py#L705) for example.

Reply:
OK. I will test it

Reply:
hello，can you supply a version of mmdet for test. mmdet v3.0.0rc5 don't contain `from mmdet.models import build_head`

Reply:
hello，can you supply a version of mmdet for test. mmdet v3.0.0rc5 don't contain `from mmdet.models import build_head` and mmcv version is 2.0.0rc3

Reply:
Seems in the latest mmdet, you can use `from mmdet.registry import MODELS` and `self.bbox_head = MODELS.build(bbox_head)`. `build_head` exists in `v3.0.0rc5`